warehouse/malware/checks/example.py

```diff
@@ -26,7 +26,11 @@ class ExampleCheck(MalwareCheckBase):
     def __init__(self, db):
         super().__init__(db)
 
-    def scan(self, file_id):
+    def scan(self, **kwargs):
+        file_id = kwargs.get("obj_id")
```

Comment:
N.B.: It's probably overkill to guard on `obj_id`, since it's explicitly supplied during `run_check`.